common/components/common/integrations/SocialMediaSignupSection.jsx

```diff
@@ -33,7 +55,7 @@ class SocialMediaSignupSection extends React.Component<{||}, State> {
   _renderSocialLoginsSection(): React$Node {
     return (
       <div className="text-center">
```

Comment:
I can't do a multiline suggest since this bit includes a deleted line, but I suggest this:
```
return (
  <React.Fragment>
    <h5 className="text-center">OR</h5>
    {this._renderSocialLogins()}
  </React.Fragment>
)
```
It does two things: Removes a div we don't need out of the DOM, and only applies text-center where it matters (aka the H5) which lets us line up our content in the SocialLogins more cleanly. Combined with the changes in the CSS to the `a` tag, it should line up both social icons and the "Continue with..." text cleanly.